lib/csv.rb

```diff
@@ -547,6 +547,21 @@ def to_csv(**options)
     end
     alias_method :to_s, :to_csv
 
+    #
+    # Extracts the nested value specified by the sequence of name objects by calling dig at each step,
+    # returning nil if any intermediate step is nil.
+    #
+    def dig(index_or_header, *index_or_headers)
```

Comment:
`*index_or_headers` isn't a good name because they are arguments for `Array#dig`. `Array` doesn't have header.
`*indexes` is better.

Reply:
Thanks, I corrected arguments name.